packages/design-system/src/components/HelpDrawer/HelpDrawer.tsx

```diff
@@ -14,29 +14,12 @@ export interface HelpDrawerProps extends DrawerProps {
    * In cases where this component’s heading may contain **sensitive information**, use this prop to override what is sent to analytics.
    */
   analyticsLabelOverride?: string;
-  /**
-   * @hide-prop [Deprecated] This prop has been renamed to `heading`.
-   */
-  title?: string | React.ReactNode;
 }
 
 export const HelpDrawer = (props: HelpDrawerProps) => {
   const { analytics, analyticsLabelOverride, children, className, ...others } = props;
   const headingRef = useHelpDrawerAnalytics(props);
 
-  if (process.env.NODE_ENV !== 'production') {
-    if (props.title) {
-      console.warn(
-        `[Deprecated]: Please remove the 'title' prop in <HelpDrawer>, use 'heading' instead. This prop has been renamed and will be removed in a future release.`
-      );
-    }
-    if (!props.title && !props.heading) {
```

Comment:
I removed this whole warning because `heading` is a required prop via TS and this doesn't seem necessary

Reply:
Yeah, that makes sense 👍